src/test/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContextPathVariableResolverTest.java

```diff
@@ -27,9 +27,9 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 
 import java.lang.reflect.Executable;
-import java.util.Arrays;
-import java.util.Collection;
+import java.util.*;
```

Comment:
avoid wildcard imports, the library uses strict import convention  and don't change it.